types.go

```diff
@@ -55,6 +55,11 @@ type (
 		// Checksum ensures the integrity of packages
 		// refereced by URL. Ignored for literals.
 		Checksum Checksum `json:"checksum"`
+	}
+
+	PackageRef struct {
+		PackageName      string
+		PackageNamespace string
```

Comment:
Does the entrypoint always point to a function? If so, just call it `fn`, `path`, `resource`, or `target`, as it seems to be just for the identification/referencing of a function/file. Handler or entrypoint sound like a command or something special going on.